reference/7.3/Microsoft.PowerShell.Core/About/about_Language_Modes.md

```diff
@@ -85,70 +187,38 @@ Only the following comparison operators are permitted:
+### ConstrainedLanguage mode
 
-**ConstrainedLanguage** mode is designed to support User Mode Code Integrity
-(UMCI) on Windows RT. It is the only supported language mode on Windows RT, but
-it is available on all supported systems.
+`ConstrainedLanguage` mode is designed to allow basic language elements such as
+loops, conditionals, string expansion, and access to object properties. The
+restrictions prevent operations that could be abused by a malicious actor.
 
-UMCI protects ARM devices by allowing only Microsoft-signed and
-Microsoft-certified apps to be installed on Windows RT-based devices.
-**ConstrainedLanguage** mode prevents users from using PowerShell to circumvent
-or violate UMCI.
+The `ConstrainedLanguage` mode permits all cmdlets and a subset of PowerShell
+language elements, but limits the object types that can be used.
 
-The features of **ConstrainedLanguage** mode are as follows:
-
-- All cmdlets in Windows modules, and other UMCI-approved cmdlets, are
-  fully functional and have complete access to system resources, except as
-  noted.
+The features of `ConstrainedLanguage` mode are as follows:
 
+- All cmdlets in Windows modules are fully functional and have complete access
+  to system resources, except as noted.
 - All elements of the PowerShell scripting language are permitted.
-
 - All modules included in Windows can be imported and all commands that the
   modules export run in the session.
-
-- The `Add-Type` cmdlet can load signed assemblies, but it cannot load
-  arbitrary C# code or Win32 APIs.
-
+- The `Add-Type` cmdlet can load signed assemblies, but it can't load arbitrary
+  C# code or Win32 APIs.
 - The `New-Object` cmdlet can be used only on allowed types (listed below).
-
-- Only allowed types (listed below) can be used in PowerShell. Other types are
-  not permitted.
-
-- Type conversion is permitted, but only when the result is an allowed type.
-
+- Only allowed types can be used in PowerShell. Other types aren't permitted.
+  Type conversion is permitted, but only when the result is an allowed type.
 - Cmdlet parameters that convert string input to types work only when the
   resulting type is an allowed type.
-
-- The `ToString()` method and the .NET methods of allowed types (listed
-  below) can be invoked. Other methods cannot be invoked.
-
+- The `ToString()` method and the .NET methods of allowed types can be invoked.
 - Users can get all properties of allowed types. Users can set the values of
-  properties only on Core types. Only the following COM objects are permitted:
+  properties only on allowed types.
 
-  - **Scripting.Dictionary**
-  - **Scripting.FileSystemObject**
-  - **VBScript.RegExp**
-
-The following types are permitted in **ConstrainedLanguage** mode. Users
-can get properties, invoke methods, and convert objects to these types.
+The following .NET types are permitted in `ConstrainedLanguage` mode. Users can
+get properties, invoke methods, and convert objects to these types.
```

Comment:
I see that the types for files and folders are missing, am I correct in understanding that I therefore couldn't do `(Get-Item ./file.yaml).LastWriteTime` then? Is this list accurate/up-to-date?

Reply:
Need to re-verify the list of types.
```
PS> $ExecutionContext.SessionState.LanguageMode = "ConstrainedLanguage"
PS> (dir .\.editorconfig).LastAccessTime

Monday, January 9, 2023 11:11:16 AM
```